torch/_higher_order_ops/wrap.py

```diff
@@ -170,11 +172,12 @@ def __call__(self, gmod, *args, **kwargs):
         from torch.fx import Interpreter
 
         if "_checkpoint_context_fn" in gmod.meta:
-            log.warning(
+            warning_once(
```

Comment:
We might want to pass `stacklevel=2`, or else the attributed line will be from `torch/_logging/_internal.py`.